zephyr/Kconfig

```diff
@@ -115,4 +115,11 @@ config WPA_SUPP_DEBUG_LEVEL
       available levels and functions for emitting the messages. Note that
       runtime filtering can also be configured in addition to the compile-time
       filtering.
+
+config WPA_SUPP_DEBUG_SHOW_KEYS
```

Comment:
Is it possible to make WPA_SUPP_LOG_LEVEL_DBG as a pre-condition for this?

Reply:
+1